src/tests/system/tests/test_proxy.py

```diff
@@ -0,0 +1,64 @@
+"""
+Proxy Provider tests.
+
+:requirement: Ldap Provider - nss-pam-ldapd
+"""
+
+
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.ldap import LDAP
+from sssd_test_framework.topology import KnownTopology
+
+
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_invalid_attribute_syntax(client: Client, ldap: LDAP):
+    """
+    :title: Enabling proxy_fast_alias shows "ldb_modify failed:
+        [Invalid attribute syntax]" for id lookups.
+    :setup:
+        1. Setup sssd for proxy provider. And enable proxy_fast_alias.
+        2. Enable proxy_fast_alias.
+        3. Setup nslcd services.
+        4. Add Ou and User.
+    :steps:
+        1. id lookup a user.
+        2. Check logs for "ldb_modify failed".
+    :expectedresults:
+        1. id look up should success.
+        2. Errors should not be seen on enabling proxy_fast_alias.
+    :customerscenario: True
+    """
+    client.sssd.import_domain("test", ldap)
+    client.sssd.restart()
```

Comment:
On line 58 we also have `client.sssd.restart()` do we need both?

Reply:
Fixed